### DISCUSSION TITLE
생성 된 캐릭터의 태그를 Play로 바꾸고 싶습니다(초보)

### DISCUSSION THREAD

**Discussion by bestdeveloperjo** [naverz/zepeto-studio-kor#1000] [5. Q&A]
export default class CharacterController extends ZepetoScriptBehaviour {
   
    public spawn:GameObject; //스폰 위치를 위한 오브젝트 변수 생성 
    private zepetocharacter : ZepetoCharacter;

    Start() 
    {        
        let a:SpawnInfo = new SpawnInfo();    // let 할당한다 새로운 스폰정보를 준다
        a.position = this.spawn.transform.position;  // a 포지션은 스폰의 위치다 
        a.rotation = this.spawn.transform.rotation;  // a 방향은 스폰의 방향이다.
        // (Editor를 통해 로그인한)UserId를 가져옵니다.  
        ZepetoPlayers.instance.CreatePlayerWithUserId("","",a, true);
        ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {            
            let _player : LocalPlayer = ZepetoPlayers.instance.LocalPlayer;
        });
     
    }            
}
여기서 생성된 캐릭터의 태그를 Play로 바꾸고 싶습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 아래와 같이 사용해 보세요.

```typescript
export default class CharacterController extends ZepetoScriptBehaviour {

public spawn:GameObject; //스폰 위치를 위한 오브젝트 변수 생성 
private zepetocharacter : ZepetoCharacter;

Start() 
{        
    let a:SpawnInfo = new SpawnInfo();    // let 할당한다 새로운 스폰정보를 준다
    a.position = this.spawn.transform.position;  // a 포지션은 스폰의 위치다 
    a.rotation = this.spawn.transform.rotation;  // a 방향은 스폰의 방향이다.
    // (Editor를 통해 로그인한)UserId를 가져옵니다.  
    ZepetoPlayers.instance.CreatePlayerWithUserId("","",a, true);
    ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {            
        let _player : LocalPlayer = ZepetoPlayers.instance.LocalPlayer;
        this.zepetoCharacter = ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character;
        // Set Player Tag
        this.zepetoCharacter.gameObject.tag = "Play";
    });
 
}            
}

```
